front/src/Main.elm

```diff
@@ -472,6 +495,19 @@ drinkIcon recipe =
         Highball ->
             collinsIcon
 
+        -- To do:
+        Margarita ->
+            cocktailIcon
+        Hurricane ->
+            cocktailIcon
+        IrishCoffeeMug ->
+            cocktailIcon
+        SteelCup ->
+            cocktailIcon
+        Wine ->
+            cocktailIcon
+        ZombieGlass ->
+            cocktailIcon
```

Comment:
Where are icons from? How can I add them?

Reply:
So… they're in Icons.elm and I have been drawing them in Figma on a 50x50px frame, then running svgo to optimize them, and then rewriting the SVG in Elm's SVG 'vdom' syntax. A little crazy :)

Reply:
(if you aren't set up with a SVG workflow you like, I can definitely draw these!)

Reply:
I could implement and optimize SVG but I can’t draw (wanting to get into that though, and been meaning to ask you about tips), so would appreciate if you could do that? Wine glass and hurricane are useful probably, margarita may be similar to standard cocktail. Zombie, steel, copper, irish coffee could maybe be grouped in one funky icon?

Reply:
Okay, I've sketched up wine, hurricane, steel, and irish coffee glasses - I'll also do a margarita glass. I think at this teensy tiny scale, a zombie glass and a collins look so similar we should group 'em as I've been doing with highball & collins.

Reply:
Erg, I'm not sure how to commit to this PR - is the 'allow maintainer edits' checkbox checked? [Here's a branch locally with the icons!](https://github.com/tmcw/flair/commit/4819cf25780cd3ed17100aaab66cdda3739aa1aa)

Reply:
Hmm yeah, it was allowed 🤷‍♂️ <img width="293" alt="Screen Shot 2020-03-19 at 7 46 20 am" src="https://user-images.githubusercontent.com/944406/77039200-baab3180-69b5-11ea-86a0-2de3497e77b0.png">

Reply:
Added your commit on top!